cpp/mrc/include/mrc/modules/mirror_tap/mirror_tap_util.hpp

```diff
@@ -0,0 +1,139 @@
+/**
+ * SPDX-FileCopyrightText: Copyright (c) 2021-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "mrc/modules/mirror_tap/mirror_tap.hpp"
+#include "mrc/modules/mirror_tap/mirror_tap_stream.hpp"
+#include "mrc/modules/module_registry_util.hpp"
+#include "mrc/modules/properties/persistent.hpp"
+#include "mrc/modules/segment_modules.hpp"
+#include "mrc/node/operators/broadcast.hpp"
+#include "mrc/segment/builder.hpp"
+#include "mrc/segment/egress_ports.hpp"
+#include "mrc/segment/ingress_ports.hpp"
+#include "mrc/version.hpp"
+
+#include <glog/logging.h>
+#include <nlohmann/json.hpp>
+
+#include <atomic>
+
+namespace mrc::modules {
+template <typename DataTypeT>
+class MirrorTapUtil
```

Comment:
I think of "Util" classes as static classes with a bunch of utility functions. I think `MirrorTapHelper` might be a better name for something you instantiate.

Reply:
Agreed. Done.